### PR TITLE
fixed the listener in the background page to return a promise

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[The error message is received on every request to dapp-connector](https://emurgo.atlassian.net/browse/YOEXT-758)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/chrome/extension/background.js [hunk 1]
```diff
@@ -939,11 +939,12 @@ chrome.runtime.onMessage.addListener(
   // Returning `true` is required by Firefox, see:
```
Comment (yushih) [inline packages/yoroi-extension/chrome/extension/background.js]: Might want to change the comment as well to be consistent with the code?

Comment (zuzunker) [reply packages/yoroi-extension/chrome/extension/background.js]: ah heck yes

**Review by yushih — APPROVED** [review]

**Review by Nebyt — APPROVED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**